tests/bson/bson-generate-document-id.phpt

```diff
@@ -48,23 +48,23 @@ array(3) {
     string(8) "New York"
   }
   ["_id"]=>
-  object(%s\ObjectID)#%d (%d) {
+  object(%s\ObjectID)#%d (1) {
     ["oid"]=>
     string(24) "%s"
   }
 }
 
 Dumping fetched user document:
-array(3) {
+object(stdClass)#%d (3) {
   ["_id"]=>
-  object(%s\ObjectID)#%d (%d) {
+  object(%s\ObjectID)#%d (1) {
     ["oid"]=>
```

Comment:
keep as %d please.
Who knows. maybe we'll add extra data to the ObjectID (like seperating out the timestamp in a var_dump()), and then changing this one is annoying (especially since it creates a weird looking failed diff until you realize whats wrong)